roles/rsyslog/defaults/main.yaml

```diff
@@ -989,32 +1017,30 @@ rsyslog__conf_viaq_elasticsearch:
                   retryfailures="on"
                   retryruleset="try_es"
                   usehttps="on"
-              {% if use_omelasticsearch_cert | default(False) %}
-                  tls.cacert="{{logging_elasticsearch_ca_cert}}"
-                  tls.mycert="{{logging_elasticsearch_cert}}"
-                  tls.myprivkey="{{logging_elasticsearch_key}}"
-              {% endif %}
-              )
-              }
+          {% if use_omelasticsearch_cert | default(False) %}
```

Comment:
Shouldn't this line and the 3 lines below it be removed since they were moved inside the loop? @nhosoi

Reply:
I think this is ok.  My test passed. :)  The diff might be unclear; the real code looks like this.
```
          {% else %}
              else if $.viaq_index_prefix startswith "{{ res.index_prefix }}" then {
                action(
                    type="omelasticsearch"
                    name="{{ res.name | default('viaq-elasticsearch') }}"
                    server="{{ res.server_host | default('logging-es') }}"
                    serverport="{{ res.server_port | default(9200) | int }}"
                    template="viaq_template"
                    searchIndex="index_template"
                    dynSearchIndex="on"
                    searchType="com.redhat.viaq.common"
                    bulkmode="on"
                    writeoperation="create"
                    bulkid="id_template"
                    dynbulkid="on"
                    retryfailures="on"
                    retryruleset="try_es"
                    usehttps="on"
            {% if use_omelasticsearch_cert | default(False) %}
                    tls.cacert="{{ res.ca_cert }}"
                    tls.mycert="{{ res.cert }}"
                    tls.myprivkey="{{ res.key }}"
            {% endif %}
                )
          {% endif %}
```